data/sql/base/db_world/reference_loot_template.sql

```diff
@@ -13953,7 +13953,7 @@ INSERT INTO `reference_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `
 	(45002, 3180, 0, 10, 0, 1, 0, 1, 1, 'Flecked Raptor Scale'),
 	(45002, 3685, 0, 60, 0, 1, 0, 1, 1, 'Raptor Egg'),
 	(45002, 5635, 0, 8, 0, 1, 0, 1, 1, 'Sharp Claw'),
-	(45002, 24065, 24065, 5, 0, 1, 1, 1, 1, '(ReferenceTable)'),
+	(45002, 24063, 24063, 3, 0, 1, 1, 1, 1, '(ReferenceTable)'),
```

Comment:
Do not edit the base SQL files.

Reply:
Ok then what should i do if it appears the problem lies in the SQL files?

Reply:
We follow this guide: https://www.azerothcore.org/wiki/sql-standards :)
In this case it should be used a DELETE + INSERT query instead. i.e. 
```
DELETE FROM `reference_loot_template` WHERE `entry`=45002 AND `item` IN (24065, 24063);
```